source/developers-guide/plugin-configuration/index.md

```diff
@@ -0,0 +1,641 @@
+    {
+        $this->subscribeEvent(
+                'Enlight_Controller_Action_PostDispatch',
+                'onPostDispatch'
+        );
+ 
+        $form = $this->Form();
+        $parent = $this->Forms()->findOneBy(array('name' => 'Frontend'));
+        $form->setParent($parent);
+        $form->setElement(
+            'checkbox', 
+            'show', 
+            array(
+                'label' => 'Plugin Anzeigen',
+                'value' => true,
+                'scope' => Shopware\Models\Config\Element::SCOPE_SHOP
+            )
+        );
+ 
+        return true;
+    }
+ 
+    public function onPostDispatch(Enlight_Event_EventArgs $arguments)
+    {
+        $config = $this->Config();
+        if (empty($config->show)) {
+            return;
+        }
+ 
+        $controller = $arguments->getSubject();
+        $view = $controller->View();
+ 
+        $view->assign('swagSubshopVisibility', 'Test text.');
+ 
+        $view->addTemplateDir($this->Path() . 'Views/');
+    }
+}
+```
+First we register to the `PostDispatch` event by using the `install()` method. The `postDispatch` event
+gets triggered after the __PostDispatch__ and will trigger the actual process. Next we add the plugin
+configuration via the `Form()` method like described earlier in this article. Later we will change the
+visibility of these configuration options related to subshops.
+```
+$form = $this->Form();
+```
+Now we have an instance of `Shopware\Models\Config\Form` in the variable `$form`. We can then assign elements with
+help of the `setElement()` method.
+```
+$form->setElement('checkbox', 'show', array(
+    'label' => 'Plugin Anzeigen',
+    'value' => true,
+    'scope' => Shopware\Models\Config\Element::SCOPE_SHOP
+));
+```
+With the help of the __scope__ parameter we can assign configurations per subshop. Without the scope parameter the configuration
+is used in all subshops. We´ve mentioned this earlier this article.
+```
+public function onPostDispatch(Enlight_Event_EventArgs $arguments)
+{
+	$config = $this->Config();
+	if (empty($config->show)) {
+	    return;
+	}
+ 
+	$controller = $arguments->getSubject();
+	$view = $controller->View();
+ 
+	$view->assign('swagSubshopVisibility', 'Test text.');
+ 
+	$view->addTemplateDir($this->Path() . 'Views/');
+}
+```
+
+Within the `postDispatch()` method we create an instance of the configuration. Afterwards we test for the just created __show__ attribute. If
+this attribute is empty the method ends and the plugin will not be executed or shown any further. If __show__ is set the template is loaded and 
+assigned via __smarty__.
+```
+{extends file="parent:frontend/index/logo-container.tpl"}
+{block name="frontend_index_logo"}
+    <div style="padding-top: 10px">
+        <h1>{$swagSubshopVisibility}</h1>
+    </div>
+{/block}
+```
+
+The __logo-container.tpl__ extends the __frontend_index_logo__ block and sets the text we used in the `bootstrap.php`. Now we can open the plugin configuration via
```

Comment:
The __logo-container.tpl__ extends from the parent logo-container.tpl and overwrites the __frontend_index_logo__ block and ....